reconcile/openshift_resources_base.py

```diff
@@ -509,6 +509,25 @@ def fetch_data(namespaces, thread_pool_size, internal, use_jump_host):
     return oc_map, ri
 
 
+def filter_namespaces_by_cluster_and_namespace(namespaces,
+                                               cluster_name,
+                                               namespace_name):
+    filtered_namespaces = []
+    if cluster_name:
+        filtered_namespaces.extend(
+            [n for n in namespaces
+             if n['cluster']['name'] == cluster_name]
+        )
+    elif namespace_name:
+        filtered_namespaces.extend(
+            [n for n in namespaces
+             if n['name'] == namespace_name]
```

Comment:
When both are provided I'd expect either an error message or to have them both used.
One might want to specify cluster AND namespace.... in this case, you're ignoring the namespace.

Reply:
what about now?

Reply:
yep, looks good, thanks!